src/utils.js

```diff
@@ -48,3 +48,8 @@ export const FORM = {
 	GROUP_FOLDER: 3,
 	APP_PASSWORD: 4,
 }
+
+export const SEARCHFROM = {
+	PROJECT_TAB: 'project-tab',
+	LINK_MULTIPLE_MODAL: 'link-multiple-modal',
```

Comment:
Maybe something like this? Right now it seems like we're linking multiple modals
```suggestion
	LINK_MULTIPLE_FILES_MODAL: 'link-multiple-files-modal',
```